airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeDestination.java

```diff
@@ -150,16 +149,16 @@ public String insertIntoFromSelectQuery(String schemaName, String srcTableName,
     @Override
     public void executeTransaction(String queries) throws Exception {
       String renameQuery = "BEGIN;\n" + queries + "COMMIT;";
-      SnowflakeDatabase.executeSync(connectionFactory, renameQuery, true, rs -> null);
+      database.execute(renameQuery);
```

Comment:
i need to double check if this is going to work or if i need to alter the snowflake jdbc instance to accept multiple statemtns

Reply:
changed the connection to allow multiple statements.